### ISSUE TITLE
Documentation for working with the explorer

### ISSUE THREAD

**Issue by toto1384** [LunarVim/lunarvim.org#252]

**Comment by LostNeophyte** [comment]
I'm working on an overview of keybindings (#297), [here's](https://github.com/LostNeophyte/lunarvim.org/blob/keybind-overview/docs/03-keybind-overview.md) the WIP version. Please let me know if you have any suggestions for improving it

**Comment by kobzar** [comment]
This is good work and all user need it. But i think this is good time to remap some default keys. 
For example move:
SPS + c => SPS + b + c
e.t.c

**Comment by aniketgm** [comment]
Agreed. Please don't change any binding now, I've become used to SPC +c and I'm sure you'll agree that many, (for e.g. #3405) don't like it when workflows are changed again and again. If anyone wish to change any keybinding, they have option to override and customize it as they want. I myself have personalized and overridden few of the original keybindings to my liking and ease.

Also, I think the issue can be closed, if the OP's @toto1384 original issue is solved. Thanks @LostNeophyte for the keybindings overview. Surely helpful for the beginners.
